src/app/core/common-components/warning-not-optimized-for-small-screen/warning-not-optimized-for-small-screen.component.scss

```
@@ -0,0 +1,25 @@
+  color: rgba(0, 0, 0, 0.87);
+  padding: 1em;
+  margin-bottom: 1em;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  @include mat-elevation.elevation(1);
+
+  span {
+    min-width: 0;
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)